libraries/ms-helper/inc/controller_board_pins.h

```diff
@@ -0,0 +1,130 @@
+#pragma once
+// Standardized controller board ports and pins
+// Defined as a GPIO Address
+
+// CAN
+#define CONTROLLER_BOARD_ADDR_DEBUG_CAN_RX \
+                                           \
+  { GPIO_PORT_A, 11 }
+
+#define CONTROLLER_BOARD_ADDR_DEBUG_CAN_TX \
+                                           \
+  { GPIO_PORT_A, 12 }
```

Comment:
Why debug CAN rx/tx? isn't it just CAN rx/tx?

Reply:
![image](https://user-images.githubusercontent.com/37023871/94751811-767eed80-0357-11eb-919e-0f318dee445a.png)